app/javascript/components/miq-about-modal.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { AboutModal } from 'patternfly-react';
+
+const SHOW_ABOUT_MODAL = '@@aboutModal/show';
+const HIDE_ABOUT_MODAL = '@@aboutModal/hide';
+const LOAD_ABOUT_MODAL = '@@aboutModal/load';
+
+const showModal = () => (dispatch, getState) => {
+  if (getState().AboutModal && getState().AboutModal.data) {
+    // If the modal has been already rendered, do not request its data twice
+    return dispatch({type: SHOW_ABOUT_MODAL});
+  }
+
+  // Request the modal data from the API
+  return API.get('/api')
```

Comment:
I think there should be some modal endpoint?

Reply:
There's no such endpoint :wink: